tests/RobotFramework/tests/tedge/diag/predefined_plugins.robot

```diff
@@ -0,0 +1,71 @@
+*** Settings ***
+Library             String
+Library             ThinEdgeIO
+
+Suite Setup         Custom Suite Setup
+Suite Teardown      Get Suite Logs
+
+Test Tags           theme:troubleshooting    theme:cli    theme:plugins
+
+
+*** Test Cases ***
+01_tedge
+    ${log_names}=    Create List
+    ...    output.log
+    ...    tedge-agent.log
+    ...    tedge-mapper-c8y.log
+    ...    tedge-mapper-collectd.log
+    ...    tedge-config-list.log
+    ...    tedge.toml
+    FOR    ${log_name}    IN    @{log_names}
+        File Size Is Not Zero    ${log_name}
+    END
+    Execute Command    diff /etc/tedge/tedge.toml /results/test/01_tedge/tedge.toml
+
+02_os
+    File Size Is Not Zero    output.log
+
+03_mqtt
+    ${log_names}=    Create List
+    ...    output.log
+    ...    tedge-mqtt-sub.log
+    ...    tedge-mqtt-sub-retained-only.log
+    FOR    ${log_name}    IN    @{log_names}
+        File Size Is Not Zero    ${log_name}
```

Comment:
We could assert the presence of some known message like `tedge-agent` registration message, or something similar.